examples/SSVEP/plot_classify_ssvep_mdm.py

```diff
@@ -0,0 +1,234 @@
+"""
+====================================================================
+Offline SSVEP-based BCI Multiclass Prediction
+====================================================================
+
+Building extended covariance matrices for SSVEP-based BCI. The
+obtained matrices are shown. A Minimum Distance to Mean classifier
+is trained to predict a 4-class problem for an offline setup.
+"""
+# Authors: Sylvain Chevallier <sylvain.chevallier@uvsq.fr> ,
+# Emmanuel Kalunga , Quentin Barthélemy, David Ojeda
+#
+# License: BSD (3-clause)
+
+# generic import
+import os
+import numpy as np
+from scipy.signal import butter, lfilter, filtfilt
+import matplotlib.pyplot as plt
+
+# mne import
+from mne import get_config, set_config, find_events, create_info
+from mne.datasets.utils import _get_path
+from mne.utils import _fetch_file, _url_to_local_path
+from mne.io import Raw, RawArray
+
+# pyriemann import
+from pyriemann.estimation import Covariances
+from pyriemann.utils.mean import mean_riemann
+from pyriemann.classification import MDM
+
+# scikit-learn import
+from sklearn.model_selection import cross_val_score, RepeatedKFold
+
+
+###############################################################################
+# Loading EEG data
+# ----------------
+#
+# The data are loaded through a MNE loader
+
+SSVEPEXO_URL = 'https://zenodo.org/record/2392979/files/'
+subject, run = 12, 1
+url = '{:s}subject{:02d}_run{:d}_raw.fif'.format(SSVEPEXO_URL,
+                                                 subject, run + 1)
+sign = 'SSVEPEXO'
+key, key_dest = 'MNE_DATASETS_SSVEPEXO_PATH', 'MNE-ssvepexo-data'
+
+# Use MNE _fetch_file to download EEG file
+if get_config(key) is None:
+    set_config(key, os.path.join(os.path.expanduser("~"), "mne_data"))
+path = _get_path(None, key, sign)
+destination = _url_to_local_path(url, os.path.join(path, key_dest))
+os.makedirs(os.path.dirname(destination), exist_ok=True)
+if not os.path.exists(destination):
+    _fetch_file(url, destination, print_destination=False)
```

Comment:
I would suggest to make a method for this. We want to keep the example short and on point. If we happens to extend the set of example we might put it somewhere else to, but for now its fine to have it here in its own function.

Reply:
Yes, this is a very good idea, it could be useful to have some functions to bridge pyriemann and moabb.

Reply:
Good idea! I added a small function, more generic to download a few samples. It could be used as a starting point for a correct helper function.